src/index.js

```diff
@@ -19,21 +19,23 @@ export default function (options = {}, callback = function noop () {}) {
   let testAttempt = 1
   let parsedOptions = parseOptions(options)
   let parser = getParser(parsedOptions.parser)
+  let plainLogger = new Logger(null)
```

Comment:
Let's just use one logger class and have `Logger` inspect `parsedOptions` that are passed to it. If you are worried about having lots of output (e.g. with `DEBUG` we could change the color of the output depending on the level).